### ISSUE TITLE
Quests that maybe should be shown in Slovenia

### ISSUE THREAD

**Issue by CTJoriginal** [streetcomplete/StreetComplete#5237]
**Does this stop have tectile paving?**
- This is quite common, especially on bus stops in cities and on a lot of train stops

**Is there tactile paving at the top and bottom of these steps?**
- this isnt quite common on regular stairs but is really common on stairs neae train stations (if it's possible to activate quests just for the vicinity of specific places, this could be a good solution)

**Is there tactile paving on this curb here?**
- this feature is really common in cities

**Is there a bicycle lane or path here? What type?**
- We do have many cycling lanes on roads, especially in cities
I found this law about cycling paths: 
http://www.pisrs.si/Pis.web/pregledPredpisa?id=PRAV11505
Section 33
- the country Metadata that defines whatever there are or aren't cycle lanes contains comment with street view pointing directly at example of cycling lane

~~**What track number is here?**~~
~~- If I understood wiki correctly, this feature shows track number on train stations~~
~~https://github.com/streetcomplete/StreetComplete/assets/48687492/5f0b7760-426c-4a8f-a2d2-20ec016a962c~~
~~We have similar track numbering on nearly all train stations~~

I think it's best if these features are disabled by default, but it really bugs me that I'm unable to map things that clearly are there and could help many blind people

I can provide further information if necessary

**Comment by mnalis** [comment]
- In my (limited) experience in Slovenia, enabling `tactile paving` quests probably make sense 
- `Is there a bicycle lane or path here? What type?` can you be more specific what you think needs changing? This line https://github.com/streetcomplete/countrymetadata/blob/2b2011b6fe79eaad0f18c3a5492bf373ac9c4430/data/exclusiveCycleLaneStyle.yml#L127 ?
- `What track number is here?` - Unless I'm wrong, isn't that [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) quest which should be reported [there](https://github.com/Helium314/SCEE/issues)?

_(also, it is preferable to have **one-issue-per-request**. Ok, multiple tactile paving quests perhaps could be bundled together as they are very similar, but not all different issues like cycleways, trains etc. Because, it makes it hard to track - e.g. one might do a PR that would enable tactile pavings in Slovenia, but unless **all** the different suggestions in the issue are resolved, the issue cannot be closed, thus it would become big and hard to follow with several different discussion interspersed)_

**Comment by CTJoriginal** [comment]
Okay, I'll open multiple issues next time

- `Is there a bicycle lane or path here? What type?`
  Basically, bike lane quest is disabled even though bike lanes are fairly common
![image](https://github.com/streetcomplete/StreetComplete/assets/48687492/ac7b372b-301c-4007-a714-1f6dee473cd4)
  I was talking about this line https://github.com/streetcomplete/countrymetadata/blob/2b2011b6fe79eaad0f18c3a5492bf373ac9c4430/data/hasAdvisoryCycleLane.yml#L131

- `What track number is here?`
  Yeah you are right, I forgot to check in regular SC app, I'll report that to SCEE repo

**Comment by westnordost** [comment]
> I was talking about this line https://github.com/streetcomplete/countrymetadata/blob/2b2011b6fe79eaad0f18c3a5492bf373ac9c4430/data/hasAdvisoryCycleLane.yml#L131

I don't think so. That line is for whether the concept of an **advisory** cycle lane as opposed to an **exclusive** cycle lane exists.

---

For each point where you haven't done this yet, would you be able to provide some proof, like street view photos and/or the result of an overpass query?

**Comment by CTJoriginal** [comment]
- **Advisory lanes:**
I think road like this fits the definition of advisory bike lane:
https://www.google.com/maps/@46.156613,14.2989235,3a,60y,46.79h,78.55t/data=!3m6!1e1!3m4!1s0pxz07LwqGoYMlJWdu3Ogw!2e0!7i16384!8i8192?entry=ttu
But at least I didn't see them really commonly, thats true.

- **Tectile paving:**
For following features I have collected samples from all over the country to show these features are common
On train stop: https://www.google.com/maps/@46.1222875,14.4389981,3a,44y,154.34h,79.79t/data=!3m6!1e1!3m4!1sA9NDRWBRlPOcIxyrad_-IA!2e0!7i16384!8i8192?entry=ttu
On bus stop: https://www.google.com/maps/@46.1669785,14.314157,3a,47.7y,185.07h,81.36t/data=!3m6!1e1!3m4!1sDK00jkQphlHi1z66wEX2rA!2e0!7i16384!8i8192?entry=ttu
On road crossing: https://www.google.com/maps/@46.6580014,16.1688547,3a,75y,62.65h,56.63t/data=!3m6!1e1!3m4!1sTr5SpHfRd2jw0o1eEGeoIw!2e0!7i16384!8i8192?entry=ttu
On street: https://www.google.com/maps/@46.0523299,14.5034576,3a,31.8y,102.91h,70.58t/data=!3m6!1e1!3m4!1sHAngw91hG7PSrkhaA1x1mg!2e0!7i13312!8i6656?entry=ttu
On steps (like I said, there are really common in nearbary of train stations but I have seen them elsewhere too): https://www.google.com/maps/@46.0371435,14.4641877,3a,50.5y,337.88h,81.95t/data=!3m6!1e1!3m4!1sKqwMNKIdLLWZYflekbKyOg!2e0!7i16384!8i8192?entry=ttu

> But you know that there is a bike path overlay, right? With this, you can add and edit the bike path infrastructure without using quests.

Yeah I do, but I think if we enable these quests we could boost mapping of those lanes because you are asked to solve lane quest while completing other related quests, as opposed to overlay, where you have to focus only on mapping of lanes.
In short, from my experience you are more effective with quests than with overlay.

> Also, your screenshot does not show StreetComplete, but a different app.

It is from SCEE, but it shows same message in regular SC

**Comment by westnordost** [comment]
> I think road like this fits the definition of advisory bike lane:
> https://www.google.com/maps/@46.156613,14.2989235,3a,60y,46.79h,78.55t/data=!3m6!1e1!3m4!1s0pxz07LwqGoYMlJWdu3Ogw!2e0!7i16384!8i8192?entry=ttu
But its true that they are not so common.

Hmm, true. But maybe this is just a regular bike lane? (I.e. for all bike lanes in Slovenia it is allowed for cars to cross into bike lanes if they need to)

**Comment by CTJoriginal** [comment]
Appearantly it's allowed [[source](https://vozimse.si/cpp/pojasnjujemo-crtkana-crta-ob-robu-vozisca/)]

Translated summary:
> - If there is no other vechile on the road you drive in the middle
> - If another car drives towards you, you have to avoid it by driving over bike lane
> - If another car drives towards you and there is biker on bike lane, you have to drive on bike lane but slow down behind the biker until its safe to return to car lane

**Comment by mnalis** [comment]
> Appearantly it's allowed [[source](https://vozimse.si/cpp/pojasnjujemo-crtkana-crta-ob-robu-vozisca/)]

But it is only talking about [that](https://www.google.com/maps/@46.156613,14.2989235,3a,60y,46.79h,78.55t/data=!3m6!1e1!3m4!1s0pxz07LwqGoYMlJWdu3Ogw!2e0!7i16384!8i8192?entry=ttu) specific case of cycle lane being separated with _"dashed white line"_, right @CTJoriginal? (the article title is _"Črtkana črta ob robu vozišča"_); not for _all_ cycle lanes?

E.g. when there is **full white line** separating cycling line from the rest of the road (like [here](https://www.mapillary.com/app/?lat=46.240333100002&lng=15.625862899999&z=17&focus=photo&pKey=463063828578195&x=0.4904693423812525&y=0.5631010193950637&zoom=0)) I believe it is not allowed for cars to use cycle lane? (at least that would be the case in Croatia for the same markings, AFAIK - but I'm not the expert in Slovenian road laws; just a casual user).

**Comment by CTJoriginal** [comment]
Yes, exactly. 
After reading that article more in depth, I found out it's a relatively new thing on our road and that we might see more of those lanes in the future.

On cycle lanes marked with full line (not dashed), motor vechiles shouldn't cross them. Sometimes, that's even additionally marked with tiny bumps on white line separating cycle lane from driving lane.

**Comment by westnordost** [comment]
I see. So, would you say that both advisory and exclusive cycle lanes as described in this wiki article exist in Slovenia?

https://wiki.openstreetmap.org/wiki/Key:cycleway:lane

If so, maybe you could add Slovenia here https://github.com/streetcomplete/countrymetadata/blob/master/data/hasAdvisoryCycleLane.yml and give the source (to the legislation text and to this ticket)?

**Comment by CTJoriginal** [comment]
Yes, we in fact have all three types of lanes mentioned in wiki (except foursth style). 
I created a [pull request](https://github.com/streetcomplete/countrymetadata/pull/19) with discussed updates.

What about tectile quests?

**Comment by westnordost** [comment]
Great, thank you for your patience and your PRs! It looks like this issue can be closed now.
